### ISSUE TITLE
GET /source/<project>/<package> route does not obey the directory.xsd schema

### ISSUE THREAD

**Issue by dcermak** [openSUSE/open-build-service#9187]
# Issue Description

The `GET /source/<project>/<package>` route should obey the directory.xsd schema, i.e. the top-element `<directory>` can only have the following attributes:
- rev
- srcmd5
- count

But for instance requesting that route for Virtualization:vagrant/ruby2.6, I get: 
``` ShellSession
$ osc api /source/Virtualization:vagrant/ruby2.6
<directory name="ruby2.6" rev="1" vrev="1" srcmd5="f71d3715b36d6a5100ada5c7ede53032">
  <linkinfo project="devel:languages:ruby" package="ruby2.6" srcmd5="79e1007e6c69fc2b2161a462fd619346" xsrcmd5="06c51075393fe15fe340289afec7f2ed" lsrcmd5="f71d3715b36d6a5100ada5c7ede53032"/>
  <entry name="_link" md5="fc0145177935b00fba3049952abaa962" size="454" mtime="1530465319"/>
</directory>
```
which contains the additional attributes "vrev" and "name".

Expected Result
============

The additional attributes should either not appear or be documented in the directory.xsd.

How to Reproduce
===============
1. call `osc api /source/Virtualization:vagrant/ruby2.6`

**Comment by dcermak** [comment]
This should be fixed by #8930

**Comment by dcermak** [comment]
fixed by #8930
